packages/gcf-utils/test/gcf-bootstrapper.ts

```diff
@@ -1,4 +1,4 @@
-// Copyright 2019 Google LLC
+// Copyright 2020 Google LLC
```

Comment:
Since this file is a git mv in history, you should keep the 2019

Reply:
Got it - I wasn't sure if a move counted as a new file for the license checker bot or not

Reply:
If the bot flags it, then the bot is wrong and we can fix